go/pkg/tool/embeddedtool.go

```diff
@@ -29,6 +30,7 @@ var (
 func RegisterFlags() {
 	flag.StringVar(&inputDigest, "digest", "", "Digest in <digest/size_bytes> format.")
 	flag.StringVar(&pathPrefix, "path", "", "Path to which outputs should be downloaded to.")
+  flag.StringVar(&symlinkBehavior, "symlink_behavior", "unspecified", "For upload_dir: how to handle symlinks. One of 'unspecified', 'resolve', 'preserve'.")
```

Comment:
Let's set the current behavior as default.

Reply:
I think this is the current behavior? The original call in `ComputeMerkleTree` did not pass any value for the `SymlinkBehavior` property of the `command.InputSpec` which I _thnk_ means that the default enum value will be assigned which would be the `iota` case (`UnspecifiedSymlinkBehavior`) here? https://github.com/bazelbuild/remote-apis-sdks/blob/719a5dd43ab6b3f125211b216c48c1e8ec1b4354/go/pkg/command/command.go#L56

Reply:
Right, but that value is not handled [specifically](https://github.com/bazelbuild/remote-apis-sdks/blob/719a5dd43ab6b3f125211b216c48c1e8ec1b4354/go/pkg/client/tree.go#L94) by the client, so as is with your new validation this would mean the flag now becomes required, while it didn't use to be. So resolve should be the default to keep the current behavior, I think.

Reply:
Discussed offline and I had misunderstood. The additional of `log.Exitf` was confusing and not correct, so removed.